src/lib.rs

```diff
@@ -73,7 +73,7 @@ enum AaConfig {
 
 /// Configuration of antialiasing. Currently this is static, but could be switched to
 /// a launch option or even finer-grained.
-const ANTIALIASING: AaConfig = AaConfig::Area;
+const ANTIALIASING: AaConfig = AaConfig::Msaa16;
```

Comment:
Do we want to make `Msaa16` the default yet? Perhaps this should remain as set to area for now as we may not want to get hit by the performance cost of increased workgroup memory in Msaa16 mode.

That said, I think I would like to make this configurable at runtime. I think a reasonable approach is to introduce two levels of settings: a `RendererOptions` setting that tells the library which pipeline variants to compile/enable and a `RenderParams` setting that selects a particular AA mode (which can eventually have a "dynamic" option if we decide to support that). That way we can run the test suite in various modes without having to re-compile the code.

If you agree with that abstraction, I'd be happy to wire that up in a follow-up.

Reply:
I think it's good making `Msaa16` the default at least temporarily, if for no other reason than to exercise it. I also agree that it should be configurable, but the mechanism to do that is nontrivial (thanks for agreeing to take it on!) and seems like it can be done separately from this PR.